File: Hosts/Go/ToPython3/libraries/python-string-utils/Main_test.go
package pythonStringUtils

import (
	"fmt"
	"testing"
	"pythonStringUtils/string_utils"
)

func TestMain(t *testing.T){

	string_utils.MetaFFILoad("string_utils_MetaFFIGuest")
	//defer string_utils.Free()

	isJson, err := string_utils.Is_Json("[1, 2, 3")
	if err != nil{ t.Fatal(err) }
	fmt.Printf("[1, 2, 3? %v\n", isJson)

	isJson, err = string_utils.Is_Json(`{ "a": 1 }`)
	if err != nil{ t.Fatal(err) }
    fmt.Printf("{ \"a\": 1 }? %v\n", isJson)

	stripped, err := string_utils.Strip_Html(`"test: <a href="foo/bar">click here</a>"`, true)
	if err != nil{ t.Fatal(err) }

	fmt.Printf("Before: %v\n", `"test: <a href="foo/bar">click here</a>"`)
	fmt.Printf("After: %v\n", stripped)
}

File: Hosts/Kotlin/ToPython3/libraries/python-string-utils/Main_test.kt
import openffi.*;
fun main() {
	var strUtils = python_string_utils();
    var resTrue = strUtils.is_json("[1, 2, 3]");
    System.out.println("[1, 2, 3]? "+resTrue);
    var stripped = strUtils.strip_html("\"test: <a href=\"foo/bar\">click here</a>\"");
    System.out.println("Before: \"test: <a href=\"foo/bar\">click here</a>\"");
    System.out.println("After: "+stripped+"\n");
}